packages/cli/src/commands/capabilities.ts

```diff
@@ -173,13 +171,12 @@ export async function getStandard(this: APICommand): Promise<CapabilitySummaryWi
 }
 
 export async function getIdFromUser(this: APICommand & { primaryKeyName: string },
-		items: CapabilitySummaryWithNamespace[]): Promise<CapabilityId> {
+		items: (CapabilitySummaryWithNamespace & { [key: string]: unknown })[]): Promise<CapabilityId> {
```

Comment:
Do we need another type?

Reply:
I wouldn't think so for this one of use case but you got me looking at the code again...the only reason this was here was to make `this.primaryKeyName in item` work but since. This function was adapted from a much more generic version in in io-commands but here, `id` can just be hard-coded which makes the code simpler and solves the type error.

I've updated the code to just use `id`. In testing it, I noticed a bug in `capabilities:delete` but I'm going to fix that in another issue because I think the project is far enough along I need to stop doing more than one thing in a single issue, no matter how small. :-)